swarm/storage/localstore/subscription_pull.go

```diff
@@ -36,6 +42,8 @@ import (
 // Make sure that you check the second returned parameter from the channel to stop iteration when its value
 // is false.
 func (db *DB) SubscribePull(ctx context.Context, bin uint8, since, until uint64) (c <-chan chunk.Descriptor, stop func()) {
+	metrics.GetOrRegisterCounter(fmt.Sprintf("localstore.SubscribePull.bin%v", bin), nil).Inc(1)
```

Comment:
I think this will explode the number of measurements. Not sure we want to have the `bin` as part of the name - Grafana won't handle 255 measurements per host very well I think.

Reply:
I think we should start with a global counter for all bins, and if this is not helpful, we should think how to add granularity.

Reply:
Well, I gave my best to add metrics with the information that you provided. I am sorry for this mistake. Will fix it.

Reply:
It's fine, I am just thinking from a perspective of how we are going to make sense of these measurements, and when a measurements explodes due to such an index, generally it is very difficult to make sense of it.

If this is `very important information`, maybe logs will help with it, I don't know just yet.